byte-buddy-dep/src/test/java/net/bytebuddy/implementation/auxiliary/MethodCallProxyTest.java

```diff
@@ -81,8 +82,12 @@ public void testNonGenericParameter() throws Exception {
         Class<?> auxiliaryType = proxyOnlyDeclaredMethodOf(GenericType.class);
         assertThat(auxiliaryType.getTypeParameters().length, is(0));
         assertThat(auxiliaryType.getDeclaredMethod("call").getGenericReturnType(), is((Type) Object.class));
-        assertThat(auxiliaryType.getDeclaredFields()[1].getGenericType(), is((Type) Object.class));
-        assertThat(auxiliaryType.getDeclaredFields()[2].getGenericType(), is((Type) Number.class));
+        Vector<Type> t = new Vector<Type>();
+        for(Field x: auxiliaryType.getDeclaredFields()){
```

Comment:
I don't think iterating over auxilaryType.getDeclaredFields is necessary, if you can just use it in the assert statements (for instance using ContainsInAnyOrder) without actually using a vector would be helpful